Source/PeleLMeX.H

```diff
@@ -32,6 +32,10 @@
 #include <AMReX_ErrorList.H>
 #include <AMReX_VisMF.H>
 
+#ifdef PELELM_USE_RAD
+#include <PeleLMRad.hpp>
```

Comment:
we have `hpp` files? hmmm we are going to have to clean that up at some point (later)